### DISCUSSION TITLE
IP handling

### DISCUSSION THREAD

**Discussion by StefanKellerAC** [gerhardf1/F3F-Timer#3] [Ideas]
first: thanks for the App!

It would be handy to store the IPs for the next start of the app.
Our Router gives the PiCams fixed IPs instead of DHCP.

Another possibility would be to be able to type them in.

You could also connect when a PiCam is "beeping", sending 'Event'.

With one of these possibilities, you can connect later, without rebooting the cams.

Stefan

**Comment by gerhardf1** [comment]
Hello,
thanks for your suggestions.

I released V0.9 which stores now the IP addresses.

Gerhard
